Components/badges/badges.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/installGroot.html/install.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Badges</title>
+</head>
+<body>
+    <div class="groot">
+        <div class="groot-container first">
+            <nav>
+            <img src="/images/groot.png" style="width: 4rem;height: auto; margin: 0 0.5rem "/>
+            <span style="display: inline-flex;"><a href="/" class="square">GrooT UI</a></span>
+            <div class="version"> v1.0</div>
+            </nav>
+            <div class="item-container">
+                <div class="install-ui"><a href="/installGroot.html/install.html" class="install-text"> Installation </a></div>
+                        <a href="/Components/avatar/avatar.html" class="avatar-text ">Avatars</a>
+                        <a href="/Components/alert/alert.html" class="avatar-text">Alert</a>
+                        <a href="/Components/badges/badges.html" class="avatar-text">Badges</a>
+                        <a href="/Components/buttons/button.html" class="avatar-text">Buttons</a>
+                        <a href="/Components/card/card.html" class="avatar-text">Cards</a>
+                        <a href="/Components/modal/modal.html" class="avatar-text">Modals</a>
+                        <a href="/Components/input/input.html" class="avatar-text">Input</a>
+                </div>
+                </div>
+        <div class="groot-container second">
+            <div class="avatar-container">
+                <span style="font-size: 2.5rem;">Badges</span><br/>
+                <span style=><p style="font-size: 1.3rem; color: grey;border-left: 3px solid #6B7280;margin-left: 1rem;">Badges are basically used for labelling components.</p><p style="margin-top: 2rem; font-size: 1.3rem;color: grey;">Badges comes with three different color <span style="color: #065F46;"> green, </span> <span style="color: #fa8484;"> pink </span> & <span style="color: #78350F;"> brown </span></p></span>
+            </div>
+            
+            <div class="avatar-box">
+            <div class="badges">
+                <span class="badge-primary">
+                    Primary
+                </span>
+                <span class="badge-secondary">
+                    Secondary
+                </span>
+                <span class="badge-new">
+                    New 
+                </span>
```

Comment:
You could increase the padding for the colors of the badges for them to look more breathable.